COLLAB_RULES.md

```diff
@@ -0,0 +1,28 @@
+# General rules to ensure productivity:
+
+This document includes general rules to avoid the lack of productivity we have seen in the past 6 weeks. Any member who wishes to collaborate must adhere to these rules or find something else to work on.
+
+### 1- One commit per day:
+
+At a minimum, a collaborator must at least contribute with 1 commit per day. A commit should be meaningful, meaning that while a collaborator doesn't have to finish an entire feature every day, although that would be ideal, they should at least **add some logic to the code**. Do not forget to keep the commits as **focused and granular** as possible.
+
+### 2- One [WiP] pull request at any given time:
+
+A collaborator must have one PR with a `[WiP]` status open at any given time. It is not acceptable to have less than one, and is also not acceptable to have more than one. Work must be focused, and therefore, each collaborator must work on one thing only at a time. When the feature is finished, the status changes to `awaiting review`, and another pull request may be open.
+**Note:** A [WiP] PR is opened with the first commit on a branch.
+
+### 3- PRs with the `awaiting review` label must take priority over current task:
+
+In order to Maximize our efficiency and be able to accurately measure our progress and productivity, a PR that awaits review must be reviewed as soon as possible. A PR requires 3 reviews to be merged, however, all collaborator must review the PR within a maximum of 48 hours. This is essential to maximize our collective learning and to maximize our efficiency and ability to reflect on our progress and adapt our workflow accordingly.
```

Comment:
Instead of `A PR requires 3 reviews to be merged`, I'd prefer it like this `A PR requires at least half of collaborators reviews to be merged` after all we wouldn't know how many active collaborators we are.

Reply:
lol, okay.